README.md

```diff
@@ -1,2 +1,164 @@
-# orleans-templates
-Templates for build Microsoft Orleans based solutions
+[![NuGet](https://img.shields.io/nuget/v/Microsoft.Orleans.Templates.svg?style=flat)](http://www.nuget.org/profiles/Orleans)
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dotnet/orleans?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+# Overview
+
+This project contains the Project to make it easier to getting started with [Orleans](https://github.com/dotnet/orleans).
```

Comment:
`This projects contains templates for getting started with [Orleans](https://github.com/dotnet/orleans).`